### DISCUSSION TITLE
Openidc el7 version rpm not available

### DISCUSSION THREAD

**Discussion by sachinsharma7788** [OpenIDC/mod_auth_openidc#1112] [Q&A]
Hi All,

We have found below rpm is now not available after Thursday (14-sept-2023).
Can you please help in checking for any specific reason of deletion please and how to progress without same.

https://github.com/zmartzone/mod_auth_openidc/releases/download/v2.4.11.2/mod_auth_openidc-2.4.11.2-1.el7.x86_64.rpm

Regards,
Sachin Sharma

**Comment by dwwarr** [comment]
Asking this question as well.  We have been using mod_auth_openidc-2.4.13.2-1.el7.x86_64.rpm.  I only see an el7 RPM available for the latest mod_auth_openidc release (currently 2.4.14.3).  Was that done on purpose with the releases URL change from github.com/zmartzone/... to github.com/OpenIDC/?   Thank you.

**Comment by aura-herrera-sonarsource** [comment]
Same question here. Could you please let us know?

**Comment by zandbelt** [comment]
this is on purpose: we are providing el7 RPMs only for the latest release of mod_auth_openidc until el7 is EOL by june 2024

**Reply by dwwarr** [reply]
Thank you for the response.  I just noticed yesterday on your release page that you cannot get el7 RPMs for releases other than the latest without a commercial license.  I missed that at first when I asked my question.
